selfdrive/car/subaru/values.py

```diff
@@ -245,6 +252,13 @@ class CAR(Platforms):
       whitelist_ecus=[Ecu.fwdCamera],
       bus=0,
     ),
+    Request(
+      [SUBARU_ALT_VERSION_REQUEST],
+      [SUBARU_ALT_VERSION_RESPONSE],
+      whitelist_ecus=[Ecu.fwdCamera],
```

Comment:
@jnewb1 would you want all ECUs?

Reply:
f100 is manufacturer specific, so might have information that would be useful for fuzzy fingerprinting

Reply:
It's not in https://github.com/commaai/panda/blob/4b6f6ac1629d001ced6e86f43579d399230af614/python/uds.py#L9 which are standard. This came from:

https://github.com/commaai/panda/blob/4b6f6ac1629d001ced6e86f43579d399230af614/examples/query_fw_versions.py#L42-L43